src/image/Mask.cc

```diff
@@ -392,7 +391,7 @@ namespace {
  */
 template<typename MaskPixelT>
 void Mask<MaskPixelT>::_initializePlanes(MaskPlaneDict const& planeDefs) {
-    pexLog::Trace("afw.Mask", 5, boost::format("Number of mask planes: %d") % getNumPlanesMax());
+    LOGLS_DEBUG("afw.Mask", "Number of mask planes: " << getNumPlanesMax());
```

Comment:
Used `LOGL_DEBUG` instead. And added another commit to rename this logger to `afw.image.Mask` along with the placeholders in tests.